client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -96,10 +97,17 @@ class PostRelativeTime extends React.PureComponent {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
 		} else if ( status === 'future' ) {
-			const scheduledTime = this.props.moment( this.props.post.date ).fromNow();
+			const moment = this.props.moment;
+			const scheduledDate = moment( this.props.post.date );
+			const now = moment();
```

Comment:
Nice! Using `sameElse` is very handy! Maybe we could use it here as well:

https://github.com/Automattic/wp-calypso/blob/3ffdca6f91a98ce4b56e4ce897453033e183cd24/client/my-sites/post-relative-time-status/index.jsx#L61

Also, `now` is not used anymore...

```suggestion
```